pkg/monitoring/metrics.go

```diff
@@ -88,23 +88,23 @@ func GetRecordRulesDesc(namespace string) []RecordRulesDesc {
 				"The number of CDI import pods with high restart count",
 				"Gauge",
 			},
-			fmt.Sprintf("count(kube_pod_container_status_restarts_total{pod=~'%s-.*', container='%s'} > %s)", common.ImporterPodName, common.ImporterPodName, strconv.Itoa(common.UnusualRestartCountThreshold)),
+			fmt.Sprintf("count(kube_pod_container_status_restarts_total{pod=~'%s-.*', container='%s'} > %s) or on() vector(0)", common.ImporterPodName, common.ImporterPodName, strconv.Itoa(common.UnusualRestartCountThreshold)),
 		},
 		{
 			MetricOpts{
 				"kubevirt_cdi_upload_pods_high_restart",
 				"The number of CDI upload server pods with high restart count",
 				"Gauge",
 			},
-			fmt.Sprintf("count(kube_pod_container_status_restarts_total{pod=~'%s-.*', container='%s'} > %s)", common.UploadPodName, common.UploadServerPodname, strconv.Itoa(common.UnusualRestartCountThreshold)),
+			fmt.Sprintf("count(kube_pod_container_status_restarts_total{pod=~'%s-.*', container='%s'} > %s) or on() vector(0)", common.UploadPodName, common.UploadServerPodname, strconv.Itoa(common.UnusualRestartCountThreshold)),
```

Comment:
could you elaborate? kube_pod_container_status_restarts_total (external metric) should always tell the truth and hence the count as well?

Reply:
It's not the regression at all, but just an issue raised by @avlitman this afternoon and fix piggybacked on this PR.

Reply:
Hi, this is this ticket: https://issues.redhat.com/browse/CNV-34071